packages/merge-styles/src/styleToClassName.ts

```diff
@@ -258,11 +260,11 @@ export interface IRegistration {
 }
 
 export function styleToRegistration(options: IStyleOptions, ...args: IStyle[]): IRegistration | undefined {
-  const rules: IRuleSet = extractRules(args, undefined, undefined, options.shadowConfig);
+  const rules: IRuleSet = extractRules(args, undefined, undefined, options.shadowConfig, options.stylesheet);
   const key = getKeyForRules(options, rules);
 
   if (key) {
-    const stylesheet = Stylesheet.getInstance(options.shadowConfig);
+    const stylesheet = options.stylesheet ?? Stylesheet.getInstance(options.shadowConfig);
```

Comment:
I _think_ you could just move this before line 263 and then you'd be able to just send `stylesheet` to `extractRules` and `extractSelector` instead ofcontinually sending `shadowConfig` as well.

Reply:
Testing this out now but it does seem to work and is a nice bit of clean up!